### PR TITLE
remove PMD and bump

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
PMD

Bumped version, removed PMD file as it was for shared

# How to test?
Zero regression

# Links
https://trello.com/c/Px4dblcG/3315-java-pmd-best-practice-usevarargs-5

# Screenshots (if appropriate):

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -23,6 +23,6 @@ dev-install-common:
 	cd ssdc-rm-common-entity-model && mvn clean install
 
 dev-update-ddl:
-	./build_groundzero_ddl.sh
+	$SAMPLE_VALIDATION_BRANCH=3315-java-pmd-best-practice-usevarargs ./build_groundzero_ddl.sh
```
Comment (liamtoozer) [inline Makefile:26]: Will need to remove this before merging 👍

Comment (liamtoozer) [inline Makefile:26]: Will need to remove this before merging 👍

File: exclude-pmd.properties [hunk 2]
```diff
@@ -2,4 +2,4 @@
 #       to this file
 
 uk.gov.ons.ssdc.common.validation.ColumnValidator=UseVarargs, ArrayIsStoredDirectly, MethodReturnsInternalArray
-uk.gov.ons.ssdc.common.validation.InSetRule=UseVarargs
\ No newline at end of file
+#uk.gov.ons.ssdc.common.validation.InSetRule=UseVarargs
```
Comment (liamtoozer) [inline exclude-pmd.properties:5]: Should still be ignoring this warning explicitly?

Comment (liamtoozer) [inline exclude-pmd.properties:5]: Should still be ignoring this warning explicitly?

**Review by AdamHawtin — APPROVED** [review]

**Review by liamtoozer — APPROVED** [review]

**Pull request merged**